packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# 47.2.0 - October 17, 2019
+
+## Fixed
+
+#### docs
+
+- Fix doc sidebar ([PR #1705](https://github.com/forcedotcom/salesforcedx-vscode/pull/1705))
+
+- Reorganize user guide section ([PR #1692](https://github.com/forcedotcom/salesforcedx-vscode/pull/1692))
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Expose the template input in project create for users working with Analytics settings ([PR #1712](https://github.com/forcedotcom/salesforcedx-vscode/pull/1712))
```

Comment:
I'm not sure if this was something we wanted to let people know about. It seemed like this was specific to WAVE and could be internal only.

Reply:
We should keep it, it's not internal only since the changes to the project command will go out as part of the new cli build.

Reply:
```suggestion
- New Analytics project when running `SFDX: Create Project` ([PR #1712](https://github.com/forcedotcom/salesforcedx-vscode/pull/1712))
```
Maybe reword it so that it's easier to understand from a user perspective.